backend/src/Designer/Program.cs

```diff
@@ -44,18 +44,17 @@
 ConfigureSetupLogging();
 
 var builder = WebApplication.CreateBuilder(args);
-
-await SetConfigurationProviders(builder.Configuration, builder.Environment);
-
-ConfigureLogging(builder.Logging);
-
-ConfigureServices(builder.Services, builder.Configuration, builder.Environment);
+{
+    await SetConfigurationProviders(builder.Configuration, builder.Environment);
+    ConfigureLogging(builder.Logging);
+    ConfigureServices(builder.Services, builder.Configuration, builder.Environment);
+}
 
 var app = builder.Build();
-
-Configure(builder.Configuration);
-
-app.Run();
+{
+    Configure(builder.Configuration);
+    app.Run();
+}
```

Comment:
Out of curiosity; what does this change mean?

Reply:
I just created 2 scopes to be more understandable where we're defining di and where pipelines etc.